src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -165,5 +243,22 @@ protected virtual void OnPropertyChanged(string propertyName)
             var handler = PropertyChanged;
             if (handler != null) handler(this, new PropertyChangedEventArgs(propertyName));
         }
+
+        protected virtual List<string> GenerateOutputParameters()
+        {
+            if (outputParameters == null)
+            {
+                outputParameters = new List<String>();
+                outputParameters.Add("none");
+            }
+            return outputParameters;
+        }
+
+        protected virtual List<Tuple<string, string>> GenerateInputParameters()
+        {
+            List<Tuple<string, string>> inputPar = new List<Tuple<string, string>>();
+            inputPar.Add(Tuple.Create("", "none"));
+            return inputPar;
+        }
```

Comment:
Added [an internal task](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6126) to fold these into `Configurations.cs`.